tools/cldr-apps/js/src/esm/cldrLoad.js

```diff
@@ -779,15 +788,29 @@ function loadAllRows(itemLoadInfo, theDiv) {
 function loadAllRowsFromJson(json, theDiv) {
   isLoading = false;
   cldrSurvey.showLoader(cldrText.get("loading2"));
-  if (!json) {
+  if (!json || json.code) {
```

Comment:
what is json.code? does it imply failure to load?

I don't understand this -- if it's right, there should be an explanatory comment

Reply:
Yes it's an error code from the payload. I can add a comment. 

I updated the loader so that rather than showing a generic error on 404 it comes here 

{ code: E_BAD_SECTION }

Reply:
I added a comment.